patterns/2-structured/document-your-guiding-principles.md

```diff
@@ -124,6 +124,7 @@ All Trusted Committers of a project are published.
 * Europace AG
 * GitHub
 * Robert Bosch GmbH
+* Siemens' Inner Source strategy is built upon the guiding principles Documentation, Workflow, Transparency, Continuous Integration and Culture which are detailed in their Inner Source portal.
```

Comment:
It feels like there's maybe a word missing there but not sure what exactly the message was. Like this?

```suggestion
* Siemens' Inner Source strategy is built upon the guiding principles for Documentation, Workflow, Transparency, Continuous Integration and Culture which are detailed in their Inner Source portal.
```

Or 

```suggestion
* Siemens' Inner Source strategy is built upon the guiding principles of Documentation, Workflow, Transparency, Continuous Integration and Culture which are detailed in their Inner Source portal.
```

Reply:
@nejch It refers to the principles listed at https://innersource.siemens.com/documentation/#success-factors. I think in that case the second version with the leading "of" would fit best, right?

Reply:
@fgreinacher ahh, I see! Yes, I think that fits :bow: